packages/next/build/webpack/plugins/font-stylesheet-gathering-plugin.ts

```diff
@@ -62,13 +67,18 @@ export class FontStylesheetGatheringPlugin {
               if (parser?.state?.module?.resource.includes('node_modules')) {
                 return
               }
-              return node.name === '__jsx'
-                ? new BasicEvaluatedExpression()
-                    //@ts-ignore
-                    .setRange(node.range)
-                    .setExpression(node)
-                    .setIdentifier('__jsx')
-                : undefined
+              let result
+              if (node.name === '__jsx') {
+                result = new BasicEvaluatedExpression()
+                // @ts-ignore
+                result.setRange(node.range)
+                result.setExpression(node)
+                result.setIdentifier('__jsx')
+
+                // This was added webpack 5.
+                result.getMembers = () => []
```

Comment:
Should this be wrapped in `if(isWebpack5)` then?

Reply:
i guess that would be a safe option.
can you give me a pointer on how to run these test in webpack 5?

Reply:
`cd packages/next && yarn add webpack@next` basically 👍

Reply:
done